relay-automation/model-catalog/render_openrouter.py

```
@@ -0,0 +1,44 @@
+        catalog_path = Path(argv[0])
+    catalog = json.loads(catalog_path.read_text())
+    rows = [r for r in catalog["aliases"] if r["target"] == "openrouter"]
+    rows.sort(key=lambda r: (-len(squash(r["match"])), r["match"]))
+    lines = [
+        f"# GENERATED from HiQS-Labs/Model-catalog v{catalog['version']} — edit the catalog, not this file.",
+        "# Legacy format (GH-120): `alias: canonical-slug`, consumed by resolve-model-alias.sh.",
+        "# Regenerate with Model-catalog's scripts/render_openrouter.py --catalog <vendored catalog.json>",
+        "#   (run from anywhere; see Model-catalog PROJECT.md Phase 1 for the drift-check recipe).",
+    ]
+    for r in rows:
+        lines.append(f"{r['match']}: {r['replace']}")
```

Ruff (0.16.3)

[warning] Use `list.extend` to create a transformed list

(PERF401)